examples/sklearn_autolog/utils.py

```diff
@@ -1,5 +1,5 @@
 import mlflow
-from mlflow import MlflowClient
+from mlflow.tracking import MlflowClient
```

Comment:
This is not broken import, `from mlflow import MlflowClient` it works:
https://github.com/mlflow/mlflow/blob/5f054814c725bb70c70b1f45cff33e030b0cc505/mlflow/__init__.py#L165

Reply:
`from mlflow import MlflowClient` doesn't work now, but will in the next version (1.28.0). @jwgwalton Can you also update other examples?

Reply:
Thanks for doing this for me @harupy , I didn't see the updates